test/e2e/toolchaincluster_test.go

```diff
@@ -55,13 +55,16 @@ func verifyToolchainCluster(t *testing.T, await *wait.Awaitility, otherAwait *wa
 
 		// then the ToolchainCluster should be ready
 		require.NoError(t, err)
-		_, err = await.WaitForNamedToolchainClusterWithCondition(toolchainCluster.Name, wait.ReadyToolchainCluster)
+		namedToolchainCluster, err := await.WaitForNamedToolchainClusterWithCondition(toolchainCluster.Name, wait.ReadyToolchainCluster)
 		require.NoError(t, err)
+		checkAbsenceOfTenantRoleLabel(t, &namedToolchainCluster)
```

Comment:
sorry but I don't understand why we should expect the _absence_ of the tenant role label? 🤔

Reply:
Since `tenant` cluster role label must be added only to `member` type clusters, I i thought that is better to check that the label is not added to the wrong type of cluster (for example to the host cluster type)